bin/deploy.sh

```diff
@@ -1,33 +1,34 @@
-#!/bin/bash
-
-set -e
+#!/usr/bin/env bash
 
 ENV=$1
-RELEASE_TAG=$2
+RELEASE_TAG=$(git rev-parse HEAD)
 APP_NAME=badger-brain
 AWS_ACCOUNT=578418881509
 AWS_REGION=eu-west-1
 ECR_REPO=$AWS_ACCOUNT.dkr.ecr.$AWS_REGION.amazonaws.com/$APP_NAME
 EB_BUCKET=elasticbeanstalk-$AWS_REGION-$AWS_ACCOUNT
 
+if [ -z "$ENV" ] || [ -z "$RELEASE_TAG" ]
+then
+  echo Usage:
+  echo "  deploy.sh ENVIRONMENT"
+  exit 1
+fi
+
+set -eu
+set -o pipefail
```

Comment:
Safer bash options. Set after the variable checking as otherwise we get an ugly error message when the user forgets to pass the argument rather than the friendlier one above.